packages/@react-spectrum/calendar/stories/RangeCalendar.stories.tsx

```diff
@@ -16,7 +16,7 @@ import {RangeCalendar} from '../';
 import React from 'react';
 import {storiesOf} from '@storybook/react';
 
-storiesOf('RangeCalendar', module)
+storiesOf('Date and Time/RangeCalendar', module)
```

Comment:
![image](https://user-images.githubusercontent.com/9661127/129979775-74160544-0a53-4c1f-9939-af0c6b3b28d3.png)
got this warning when changing the range in the `'minValue: today, maxValue: 1 week from now'`, I still need to look at the stately for range calendar to see why

Reply:
Yeah calendar refactoring is coming